### PR TITLE
ALR-239

### PR DESCRIPTION
**Pull request by Keerthana9894**

### CODE REVIEW

File: src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml [hunk 1]
```diff
@@ -210,7 +210,7 @@
         <field>Case.AccountId</field>
         <readable>true</readable>
     </fieldPermissions>
-    <fieldPermissions>
+        <fieldPermissions>
```
Comment (andreyshestakov-phsa) [inline src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml:213]: remove added spaces

File: src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml [hunk 2]
```diff
@@ -220,7 +220,7 @@
         <field>Case.Origin</field>
         <readable>true</readable>
     </fieldPermissions>
-    <fieldPermissions>
+        <fieldPermissions>
```
Comment (andreyshestakov-phsa) [inline src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml:223]: removed added spaces

File: src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml [hunk 3]
```diff
@@ -230,7 +230,7 @@
         <field>Case.Subject</field>
         <readable>true</readable>
     </fieldPermissions>
-    <fieldPermissions>
+        <fieldPermissions>
```
Comment (andreyshestakov-phsa) [inline src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml:233]: remove added spaces

File: src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml [hunk 4]
```diff
@@ -619,8 +629,8 @@
         <recordType>ActionPlan.ActionPlan</recordType>
         <visible>true</visible>
     </recordTypeVisibilities>
-<recordTypeVisibilities>
-        <recordType>Asset.Units_Record_Type</recordType>
+    <recordTypeVisibilities>
+<recordType>Asset.Units_Record_Type</recordType>
```
Comment (andreyshestakov-phsa) [inline src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml:633]: set back spaces

File: src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml [hunk 5]
```diff
@@ -643,7 +653,7 @@
         <recordType>Task.Task</recordType>
         <visible>true</visible>
     </recordTypeVisibilities>
-    <userPermissions>
+        <userPermissions>
```
Comment (andreyshestakov-phsa) [inline src-access-mgmt/main/default/permissionsets/EHIS_Read_All_Public_Sector_Objects_PS.permissionset-meta.xml:656]: remove added spaces

**Review by sheetalgodse — DISMISSED** [review]

**Comment by GandlojuVishwantha** [comment]
duplicate to PR 239

**Pull request closed without merging**